config/redirects

```diff
@@ -12,6 +12,10 @@ raw: docs/drivers/go -> https://www.mongodb.com/docs/drivers/go/current/
 
 raw: docs/drivers/java -> ${base}/java-drivers/
 
+# Rust
+
+raw: docs/drivers/rust -> https://www.mongodb.com/docs/drivers/rust/current/
+
```

Comment:
Suggestion:
One other item that might be helpful to update is the drivers/drivers section on line 215. It will still work without an update, but will cause a second redirect for anyone that somehow finds their way to docs/drivers/drivers (docs/drivers/drivers/rust/ -> docs/drivers/rust/ -> docs/drivers/rust/current)